### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by TobiTenno**
Enjoy!

### CODE REVIEW

File: src/resources/de/strings.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "languages":{
+    "en-US":{
```
Comment (mephasor) [inline src/resources/de/strings.json]: This is the same for all files. Is this not used?

File: src/resources/de/strings.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      "damage_reply":"Operator $author, the damage flowchart, at your request.",
```
Comment (mephasor) [inline src/resources/de/strings.json]: "damage_reply":"Operator $author, der Schadensgraph, wie angefragt."

File: src/resources/ru/strings.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "languages":{
+    "en-US":{
+      "damage_reply":"Оператор $author, повреждение схемы, в вашем запросе.",
```
Comment (mephasor) [inline src/resources/ru/strings.json]: damage_reply":"Оператор $author, вот граф урона, по вашему запросу.",

**Comment by TobiTenno** [comment]
none of them are set in stone, there are 2 branches controlling it, and these are all changing.

File: src/resources/de/strings.json (hunk 2 above)
Comment (TobiTenno) [reply src/resources/de/strings.json]: if you want to contribute changes, submit them on crowdin, which is where these are automatically generated from

**Comment by cholojuanito** [comment]
Do you guys need help doing the Spanish localization? I'm more than happy to help

**Comment by TobiTenno** [comment]
@cholojuanito if you'd like to

**Pull request merged**